FRAuth/FRAuth/AppIntegrity/FRAppIntegrityKeys.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  FRAppAuthKeys.swift
+//  FRAuth
+//
+//  Copyright (c) 2023 ForgeRock. All rights reserved.
+//
+//  This software may be modified and distributed under the terms
+//  of the MIT license. See the LICENSE file for details.
+//
+
+
+import Foundation
+
+public struct FRAppIntegrityKeys {
+    
+    public private(set) var appAttestKey: String
+    public private(set) var assertKey: String? = nil
+    public private(set) var keyIdentifier: String
+    public private(set) var clientDataHash: String
+    private let key = "com.forgerock.ios.appattest.keychainservice"
```

Comment:
Would it be useful to allow to customise this?

Reply:
users has the access to just to delete the key stored in keychain , i am wondering whats the advantage of customizing this . lets discuss this

Reply:
since we dont share the keys between apps , we dont let users customize this